crates/iota-e2e-tests/tests/passkey_e2e_tests.rs

```diff
@@ -37,12 +37,18 @@ use url::Url;
 struct MyUserValidationMethod {}
 #[async_trait::async_trait]
 impl UserValidationMethod for MyUserValidationMethod {
-    async fn check_user_presence(&self) -> bool {
-        true
-    }
+    type PasskeyItem = Passkey;
 
-    async fn check_user_verification(&self) -> bool {
-        true
+    async fn check_user<'a>(
+        &self,
+        _credential: Option<&'a Self::PasskeyItem>,
+        presence: bool,
+        verification: bool,
+    ) -> Result<UserCheck, Ctap2Error> {
+        Ok(UserCheck {
+            presence,
+            verification,
```

Comment:
I know it's testing purposes but what's happening here? Returning arguments?

Reply:
done!

Reply:
Fixed it!